helm-chart/splunk-kubernetes-logging/values.yaml

```diff
@@ -54,6 +54,8 @@ splunk:
     #       The file will be stored as a secret in kubernetes.
     caFile:
 
+#use existing K8s Secret
+existingSecret:
```

Comment:
Instead of using `existingSecret`, I'd prefer to use
```yaml
secret:
  create: true
  name:
```
When `secret.create` is `true`, we create the secret. And if `secret.name` is blank, we use the default name.
In this way, we keep the values consistent (with `rbac`, `serviceAccout`, etc.)